calliope/test/test_constraint_results.py

```diff
@@ -799,3 +799,50 @@ def test_location_specific_energy_cap_min_constraint(self):
         expensive_capacity1 = capacity.loc[("1", "expensive_supply")]
         assert round(expensive_capacity0, 5) >= 6
         assert expensive_capacity1 == 0
+
+
+class TestEnergyCapacityPerStorageCapacity:
+
+    @pytest.fixture
+    def model_file(self):
+        return "energy_cap_per_storage_cap.yaml"
+
+    def test_no_constraint_set(self, model_file):
+        model = build_model(model_file=model_file)
+        model.run()
+        assert model.results.termination_condition == "optimal"
```

Comment:
I'm generally more of a fan of xarray syntax, since it is cleaner in these instances. E.g. here it would be: `model.get_formatted_array("energy_cap").loc[{'techs': 'my_storage'}].sum()`

Reply:
It looks good and is more concise. But your example returns an 0D Array rather than a scalar value. What is a good way to access the scalar value? The only way I found (after stackoverflowing) is `model.get_formatted_array("energy_cap").loc[{'techs': 'my_storage'}].sum().values[()]` which is not very intuitive. Do you know a better way?

Reply:
`.item()`?

Reply:
Nice!